test/lib/kind.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+# Create Kubernetes cluster with Kind
+
+set -o errexit
+
+REPO_ROOT=$(git rev-parse --show-toplevel)
+KIND_VERSION=v0.5.1
+HELM_MODE="${HELM_MODE:-v2}"
+HELM2_VERSION=v2.14.3
+HELM3_VERSION=v3.0.0-beta.5
+
+function installKubernetes() {
+  echo ">>> Installing kubectl"
+  curl -sLO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl && \
+  chmod +x kubectl && \
+  sudo mv kubectl /usr/local/bin/
```

Comment:
This is not amenable to re-running the `test/lib/kind.sh` script  multiple times, as would be done by a developer testing the repo locally. It will continually download and re-install `kubectl` even if `kubectl` is already installed.

It would be useful to have a `is_installed` helper function that would return 0 when a required binary is installed, 1 otherwise:

Something like this would be good to put into a `test/lib/common.sh` file and source'd here:

```
is_installed() {
    local name="$1"
    if $(which $name >/dev/null 2>&1); then
        return 0
    else
        return 1
    fi
}
```

you can then gate the installation of `kubectl`, `kind` and other executables based on whether they are not already installed.

Reply:
I see no point in running the kind script for local dev. No matter what Kubernetes you may have (Docker for Mac, minikube, Kind, EKS, GKE, etc) you would export the `KUBECONFIG` and run the bats tests.

Reply:
I was executing it because you have the Helm installation coupled in the same file and I didn't have Helm installed.